35_Search_Insert_Position.py

```diff
@@ -0,0 +1,21 @@
+# T(logN) O(N)
+from typing import List
+
+
+class Solution:
+    def searchInsert(self, nums: List[int], target: int) -> int:
+        middle = 0
+        left, right = 0, len(nums) - 1
+        if nums[0] > target:
+            return 0
+        if nums[-1] < target:
+            return len(nums)
+        while left <= right:
+            middle = (left + right) // 2
```

Comment:
middle = (left+right) // 2

Reply:
@Ryuju0911 
Let me confirm whether I have to change or not.
If I change from mine into yours, I got error on vscode.
```
	"message": "missing whitespace around arithmetic operator",
```
Which way should I choose?